src/Microsoft.Identity.Web.Certificateless/AzureIdentityForKubernetesClientAssertion.Log.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Identity.Web
+{
+    public partial class AzureIdentityForKubernetesClientAssertion
+    {
+    /*       
+    // High performance logger messages (before generation).
+    #pragma warning disable SYSLIB1009 // Logging methods must be static
+            [LoggerMessage(EventId = 1, Level = LogLevel.Information, Message = "SignedAssertionFileDiskPath not provided. Falling back to the content of the AZURE_FEDERATED_TOKEN_FILE environment variable.")]
+            partial void SignedAssertionFileDiskPathNotProvided(ILogger logger);
+
+            [LoggerMessage(EventId = 2, Level = LogLevel.Information, Message = "The `{environmentVariableName}` environment variable not provided.")]
+            partial void SignedAssertionEnvironmentVariableNotProvided(ILogger logger, string environmentVariableName);
+
+            [LoggerMessage(EventId = 3, Level = LogLevel.Error, Message = "The environment variable AZURE_FEDERATED_TOKEN_FILE or AZURE_ACCESS_TOKEN_FILE or the 'SignedAssertionFileDiskPath' must be set to the path of the file containing the signed assertion.")]
+            partial void NoSignedAssertionParameterProvided(ILogger logger);
+
+            [LoggerMessage(EventId = 4, Level = LogLevel.Error, Message = "The file `{filePath}` containing the signed assertion was not found.")]
+            partial void FileAssertionPathNotFound(ILogger logger, string filePath);
+
+            [LoggerMessage(EventId = 5, Level = LogLevel.Information, Message = "Successfully read the signed assertion for `{filePath}`. Expires at {expiry}.")]
+            partial void SuccessFullyReadSignedAssertion(ILogger logger, string filePath, DateTime expiry);
+
+            [LoggerMessage(EventId = 6, Level = LogLevel.Error, Message = "The file `{filePath} does not contain a valid signed assertion. {message}.")]
+            partial void FileDoesNotContainValidAssertion(ILogger logger, string filePath, string message);
+    #pragma warning restore SYSLIB1009 // Logging methods must be static
+    */
+
+        /// <summary>
+        /// Performant logging messages.
+        /// </summary>
+        static class Log
```

Comment:
rename to Logger, as well as the class, to match our other logger classes.

Reply:
Done!